docs/source/data-analytics-pipeline.rst

```diff
@@ -19,9 +19,6 @@ Introduction
 
 .. _onedal_data_analytics_pipeline:
 
-Data Analytics Pipeline
```

Comment:
Why this heading should be deleted?

Reply:
Hi Sasha I have tagged you in the jira now :)

Reply:
thanks, please add the reason to the PR description

Reply:
Hi Sasha I added the reason to the PR. Also Kevin Putnam says in the Jira "I spoke to Moore, Benjamin D and we agreed that the best way to handle this, for now, is to remove the 2nd level heading that reads "Data Analytics Pipeline" in the rst file. I will continue to follow up to see if I can figure out exactly why this is happening. 

This is what I have done here. Kevin is now looking into this some more so I will wait for Kevin and @bdmoore1 to look at the PR to confirm! Once they confirm if this is okay or not, I will proceed with their recommendation. Thanks

Reply:
@aepanchi also to add, Kevin informed me that he believes that "title being overridden in the table of contents in index_toc.rst" hope this is clearer!